src/main/java/com/connexta/search/common/SearchManager.java

```diff
@@ -21,21 +21,21 @@
    * Indexes the resource.
    *
    * @param datasetId the id of the dataset
-   * @param mediaType the type of the dataset
-   * @param inputStream the cst {@link InputStream}
+   * @param irmUri the type of the dataset
+   * @param inputStream the dataset
```

Comment:
You're right. Right now, it is pointing to a CST resource since we haven't started indexing on irm yet. I renamed everything to say irm even though it is json.

Reply:
When will be indexing with IRM? This is just confusing now IMO.

Reply:
Next sprint

Reply:
I'll add a code comment where we do the parsing to clarify